### PR TITLE
Refactor Iperf into another inherited class of PerfServer/Client

### PR DESCRIPTION
**Pull request by wizhaoredhat**

### CODE REVIEW

File: validateOffload.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 from logger import logger
 import time
 from testConfig import TestConfig
-from iperf import IperfServer, IperfClient, EXTERNAL_IPERF3_SERVER
+from iperf import IperfServer, IperfClient, EXTERNAL_PERF_SERVER
```
Comment (SalDaniele) [inline validateOffload.py:14]: EXTERNAL_PERF_SERVER is imported from perf now

File: iperf.py [hunk 2]
```diff
@@ -68,27 +33,6 @@ def __init__(self, tc: TestConfig, ts: TestSettings):
         self.cluster_ip_addr = self.create_cluster_ip_service()
         self.nodeport_ip_addr = self.create_node_port_service(self.port + 25000)
```
Comment (SalDaniele) [inline iperf.py:34]: We need to set these in the perf class as well if there is code in the perf class that expects this fields to exist

**Review by SalDaniele — COMMENTED** [review]
LGTM besides some MyPy errors.

More generally, curious why an additional layer of abstraction between Task and Iperf is required. I support the netperf task will share a significant amount of code with Iperf that isnt already included in the base Task class?

File: validateOffload.py (hunk 1 above)
Comment (wizhaoredhat) [reply validateOffload.py]: Thanks for catching this.

File: iperf.py (hunk 2 above)
Comment (wizhaoredhat) [reply iperf.py]: Yep I agree. Made the change.

**Pull request merged**